docs/user-guide/ze-profiles.md

```diff
@@ -2,10 +2,119 @@
 
 After you install Zowe Explorer, you need to have a Zowe Explorer profile to use all functions of the extension. You can optionally activate the Secure Credential Store plug-in to securely store your credentials.
```

Comment:
We should remove the `You can optionally activate the Secure Credential Store plug-in to securely store your credentials.` sentence since it's enabled by default on V2 😋

Reply:
done

Reply:
I still see this in Files Changed `You can optionally activate the Secure Credential Store plug-in to securely store your credentials.` I think it should be removed since no more SCS plugin